Create openacc.md

sunitachandra opened the pull request:
adding description for openacc

sunitachandra committed:
Create openacc.md

adding description for openacc

bernhold commented:
Hi @sunitachandra, thanks for this and your OpenMP submissions.  After talking with @keitaTN, I'd like to suggest that these two product entries be focused specifically on the V&V suites.  Those are concrete products that we should give visibility to.  I gather that Johannes is on the hook for an entry for LLVM, which will presumably emphasize the work on the OpenMP implementation.  Will that work for you?  Thanks

keitaTN commented:
[like]  Teranishi, Keita reacted to your message:
________________________________
From: David E. Bernholdt ***@***.***>
Sent: Thursday, January 16, 2025 2:02:54 AM
To: cass-community/new ***@***.***>
Cc: Teranishi, Keita ***@***.***>; Mention ***@***.***>
Subject: [EXTERNAL] Re: [cass-community/new] Create openacc.md (PR #76)


Hi @sunitachandra<https://urldefense.us/v2/url?u=https-3A__github.com_sunitachandra&d=DwMCaQ&c=v4IIwRuZAmwupIjowmMWUmLasxPEgYsgNI-O7C4ViYc&r=RnGksEeP8Hnu8nrJCbLZAjs5T2iFaJmtp2eby7VLkxA&m=dF8cOOA8_FZUm6xiWq_Ji1FjA8vUDxvArxgstXrhvUmcYrlpCBo5pqn8MIXcTBH4&s=2AOfV01Vwq5cHM1WF3XeQP8PjAbEXCdiZzk7S9rOwOw&e=>, thanks for this and your OpenMP submissions. After talking with @keitaTN<https://urldefense.us/v2/url?u=https-3A__github.com_keitaTN&d=DwMCaQ&c=v4IIwRuZAmwupIjowmMWUmLasxPEgYsgNI-O7C4ViYc&r=RnGksEeP8Hnu8nrJCbLZAjs5T2iFaJmtp2eby7VLkxA&m=dF8cOOA8_FZUm6xiWq_Ji1FjA8vUDxvArxgstXrhvUmcYrlpCBo5pqn8MIXcTBH4&s=GVnNl7LCzkNIehNZ1OL25TSkQHujmT1XBFbuU7lxPoo&e=>, I'd like to suggest that these two product entries be focused specifically on the V&V suites. Those are concrete products that we should give visibility to. I gather that Johannes is on the hook for an entry for LLVM, which will presumably emphasize the work on the OpenMP implementation. Will that work for you? Thanks

—
Reply to this email directly, view it on GitHub<https://urldefense.us/v2/url?u=https-3A__github.com_cass-2Dcommunity_new_pull_76-23issuecomment-2D2594314938&d=DwMCaQ&c=v4IIwRuZAmwupIjowmMWUmLasxPEgYsgNI-O7C4ViYc&r=RnGksEeP8Hnu8nrJCbLZAjs5T2iFaJmtp2eby7VLkxA&m=dF8cOOA8_FZUm6xiWq_Ji1FjA8vUDxvArxgstXrhvUmcYrlpCBo5pqn8MIXcTBH4&s=_VhFcT2SYdw3gydcdx5pnA_w-RgYkf7zZPH5Lq41ZKg&e=>, or unsubscribe<https://urldefense.us/v2/url?u=https-3A__github.com_notifications_unsubscribe-2Dauth_A3N3NHGYZQYY3EY5WMMO3N32K4HM5AVCNFSM6AAAAABVF4LZ66VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDKOJUGMYTIOJTHA&d=DwMCaQ&c=v4IIwRuZAmwupIjowmMWUmLasxPEgYsgNI-O7C4ViYc&r=RnGksEeP8Hnu8nrJCbLZAjs5T2iFaJmtp2eby7VLkxA&m=dF8cOOA8_FZUm6xiWq_Ji1FjA8vUDxvArxgstXrhvUmcYrlpCBo5pqn8MIXcTBH4&s=9RgRx4TyzeIb3npxKGFNEQSUK97t5FPY8x0arO9iKS0&e=>.
You are receiving this because you were mentioned.Message ID: ***@***.***>

sunitachandra commented:
Oh yeah sure, I will update, I put it that way coz as we have been exploring/working on some LLVM projects with Johannes. But happy to keep things separate.

sunitachandra committed:
Update openacc.md

sunitachandra commented:
My changes got committed to patch-2 branch. Not sure if that is OK! If it isn't please let me if I need to do something else :)

bernhold commented:
Hi @sunitachandra, I'd like to change the `name` field to reflect the V&V suites specifically.  Something like "OpenACC Verification and Validation Test Suite" or whatever your official name is.

Also, can you add some information in the `additional_resource_links` structure to point readers to appropriate resources?  The four there (Website, Repository, Downloads, and Documentation) are *suggested* rather than required.  Fill in which ever ones make sense.   If there are other resources you'd like to point to, we can include those too.  Thanks

sunitachandra committed:
Update openacc.md

Updated the website, repo and documentation files and updated the name field as well

sunitachandra commented:
Thank you @bernhold for these suggestions! I completely overlooked, I have now updated both the files with links as well and updated the name.
